processor/resourcedetectionprocessor/README.md

```diff
@@ -190,6 +192,17 @@ able to determine `host.name`. In that case, users are encouraged to set `host.n
     * faas.name (service name)
     * faas.version (service revision)
 
+#### Cloud Run Jobs Metadata
+
+    * cloud.provider ("gcp")
+    * cloud.platform ("gcp_cloud_run")
+    * cloud.account.id (project id)
+    * cloud.region (e.g. "us-central1")
+    * faas.id (instance id)
+    * faas.name (service name)
+    * gcp.cloud_run.job.execution ("my-service-ajg89")
+    * gcp.cloud_run.job.task_index ("0")
```

Comment:
Pardon my ignorance here, but should the new resource attribute be included in this list, or are they irrelevant here?

Reply:
The attributes for cloud run jobs should have been added in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/23681, and they're enabled by default, so I do think so.

The optional `gcp.gce.instance.hostname` and `gcp.gce.instance.name` (this PR, disabled by default) I wasn't sure about, so I labeled them as optional in the readme.